docs/learn-auction.md

```diff
@@ -96,25 +96,24 @@ community.
 ```
 Parachain slots at genesis
 
-       --6 months--
+       --3 months--
        v          v
-Slot A |     1    |     2    |     3     |     4     |...
-Slot B |     1    |     2    |     3     |     4     |...
-Slot C |__________|     1    |     2     |     3     |     4     |...
-Slot D |__________|     1    |     2     |     3     |     4     |...
-Slot E |__________|__________|     1     |     2     |     3     |     4     |...
-       ^                                             ^
-       ---------------------max lease-------------------
+Slot A |     1    |     2     |     3     |     4     |     5     |     6    |     7     |     8     |     9     |...
+Slot B |     1    |     2     |     3     |     4     |     5     |     6    |     7     |     8     |     9     |...
+Slot C |__________|     1     |     2     |     3     |     4     |     5    |     6     |     7     |     8     |     9     |...
+Slot D |__________|     1     |     2     |     3     |     4     |     5    |     6     |     7     |     8     |     9     |...
+Slot E |__________|___________|     1     |     2     |     3     |     4    |     5     |     6     |     7     |     8     |     9     |...
+       ^                                                                                             ^
+       ---------------------------------------------max lease-----------------------------------------
 
 ```
 
 _Each period of the range 1 - 4 represents a
-{{ polkadot: 6-month duration for a total of 2 years :polkadot }} or a shorter duration on Kusama_
+{{ polkadot: 3-month duration for a total of 2 years :polkadot }} and a 6-week duration on Kusama_
```

Comment:
This will just say

> and a 6-week duration on Kusama

on the Kusama guide. The point of conditionals is to render A if polkadot and B if kusama.